custom/conf/app.example.ini

```
@@ -1450,6 +1450,8 @@ LEVEL = Info
 ;;
 ;; Default configuration for email notifications for users (user configurable). Options: enabled, onmention, disabled
 ;DEFAULT_EMAIL_NOTIFICATIONS = enabled
+;; Send email notifications to all instance admins on new user sign-ups. Options: enabled, true, false
```

```suggestion
;; Send an email to all admins when a new user signs up to inform the admins about this act. Options: true, false
```
Additionally, if you change the example config, please also document this in `docs/content/administration/config-cheat-sheet.en-us.md`